test/integration/test_seq_copy_int.py

```diff
@@ -45,23 +46,23 @@
      " --checkpoint-frequency 2 --optimizer adam --initial-learning-rate 0.01 --batch-type sentence "
      " --decode-and-evaluate 0",
      "--beam-size 2 --softmax-temperature 0.01",
-     False, False),
+     False, False, False),
```

Comment:
so now all tests disable the check against scores?

Reply:
Yes, disabled after a discussion that we don't need to ensure their closeness in integration tests and because they fail sporadically. Added a reminder in a TODO.